plugins/postcss-nesting/README.md

```diff
@@ -80,43 +82,28 @@ instructions for:
 - [Gulp](INSTALL.md#gulp)
 - [Grunt](INSTALL.md#grunt)
 
-## ⚠️ `@nest` has been removed from the specification.
-
-Previous iterations of the [CSS Nesting specification] required using `@nest` for certain selectors.
-
-`@nest` was removed from the specification completely.
-Future versions of this plugin will error if you use `@nest`.
-
-We advice everyone to migrate their codebase **now** to nested CSS without `@nest`.  
-We published a [Stylelint Plugin](https://github.com/csstools/postcss-plugins/tree/main/plugins-stylelint/no-at-nest-rule#csstoolsstylelint-no-at-nest-rule) to help you migrate.
-
-example warning:
-> `@nest` was removed from the CSS Nesting specification and will be removed from PostCSS Nesting in the next major version.
-> Change `@nest foo & {}` to `foo & {}` to migrate to the latest standard.
-
-You can silence this warning with a new `silenceAtNestWarning` plugin option.
-
-```js
-postcssNesting({
-	silenceAtNestWarning: true
-})
-```
-
 ## Options
 
 ### edition
 
-The default behavior is to transpile CSS following an older version of the CSS nesting specification.
+The default behavior is to transpile CSS following the version of the CSS nesting specification as it existed during the last major version of this plugin.
```

Comment:
I'm not sure this sentence is clear enough.

Is there more recent work to the spec? While I know we don't need to explain the specs I feel it doesn't hurt to be more explicit here.

Reply:
I was being vague on purpose but I see know that we need to be clearer on why this is vague.

I didn't want to have to change this sentence each time we ship a new edition or change the default.

Maybe better to explain why we have this mechanic instead.

Reply:
@Antonio-Laguna I've rewritten this section which more focus on what it is and when users should set this option. Hopefully an improvement :)

https://github.com/csstools/postcss-plugins/blob/postcss-nesting-make-2024-the-default--sympathetic-spadefoot-toad-3bea5e0b0d/plugins/postcss-nesting/README.md#edition